src/test/java/divaeva/hw/api/store/GetStoreInventoryTest.java

```diff
@@ -0,0 +1,21 @@
+package divaeva.hw.api.store;
+
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import rest.client.RestClient;
+
+import java.util.Map;
+
+public class GetStoreInventoryTest extends StoreTestBase {
+
+    private RestClient<Map> restClient = new RestClient<>(BASE_SERVER_URL_STORE_INVENTORY);
+
+    @Test
+    public void getStoreInventoryTest() {
+        Response getResponse = restClient.getEntity();
```

Comment:
I can't find DTO for this response

Reply:
I have verified this with swagger API and response here is a Map
<img width="517" alt="Screenshot 2023-10-14 at 23 36 16" src="https://github.com/AnastasiaDivaeva/API_HW/assets/139628199/661ffc7b-5f8f-4fae-970a-eaffcdf93e19">
A few days ago response here was different, so I cannot create a DTO for this response due to dynamically changing content.